index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>John Thiong'o Murianki</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap' rel='stylesheet'>
+</head>
+<body>
+    <div id ="content">
+    <div id ="status-bar"></div>
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you remove the are showing the time, battery icon signal icon and wifi icon.

<img width="318" alt="image" src="https://user-images.githubusercontent.com/52934128/213402489-8e0e6b3e-27dc-4e84-ba42-c33a56d19e08.png">